FixDisk/readme.md

```diff
@@ -0,0 +1,35 @@
+# Overview
+Occasionally Azure IaaS VMs may not start because there is something wrong with the operating system (OS) disk preventing it from booting up correctly.
+In such cases it is a common practice to recover the problem VM by performing the following steps:
+
+- Delete the VM (but keep the disks)
+- Attach the disk(s) to another bootable VM as a Data Disk
+- Run the script https://github.com/sebdau/azpstools/blob/master/FixDisk/TS_RecoveryWorker2.ps1 as an elevated administrator from the recovery VM
```

Comment:
can we fix the url to point to azure support scripts?  might be easiest to put a relative hyperlink like [TS_RecoveryWorker2.ps1](TS_RecoveryWorker2.ps1)

Reply:
done